app/app.h

```diff
@@ -9,6 +9,33 @@ namespace vsite::oop::v2
 /*
 * class declaration
 */
+    std::string to_hex(int value);
 
 
+    class color {
+    private:
+        double red;
+        double green;
+        double blue;
+
+    public:
+        color();
+
+        void set_red(double r);
+        void set_green(double g);
+        void set_blue(double b);
+
+        double get_red() const;
+        double get_green() const;
+        double get_blue() const;
+
+        color(const color& other);
+
+        color& operator=(const color& other);
```

Comment:
`operator=`? Otkud to, to radimo tek za mjesec dana...

Reply:
Izbacivalo mi je čudnu grešku u test.obj i test.dll i nisam znao zašto pa sam pitao prijatelja za pomoć

Reply:
Ne treba vam `operator=` za klasu `color.` Pošaljite koju ste to grešku dobivali bez njega.

Reply:
![Screenshot 2024-11-04 135008](https://github.com/user-attachments/assets/f3b69592-0e1f-42d2-af7d-3b7145fe3707)